src/test/java/org/pesmypetcare/webservice/service/MedicationServiceTest.java

```diff
@@ -0,0 +1,152 @@
+package org.pesmypetcare.webservice.service;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.pesmypetcare.webservice.dao.MedicationDao;
+import org.pesmypetcare.webservice.entity.MedicationEntity;
+import org.pesmypetcare.webservice.error.DatabaseAccessException;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertSame;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.isA;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+public class MedicationServiceTest {
+    private static List<Map<List<String>, Object>> MedicationList;
+    private static MedicationEntity MedicationEntity;
+    private static String owner;
+    private static String petName;
+    private static String dateName;
+    private static String date;
+    private static String date2;
+    private static String field;
+    private static int value;
+
+    @Mock
+    private MedicationDao MedicationDao;
+
+    @InjectMocks
+    private MedicationService service = new MedicationServiceImp();
+
+    @BeforeAll
+    public static void setUp() {
+        MedicationList = new ArrayList<>();
+        MedicationEntity = new MedicationEntity();
+        dateName = "2020-02-13/Cloroform";
+        date = "2020-02-13";
+        date2 = "2021-02-13";
+        owner = "Pepe05";
+        petName = "Camper";
+        field = "quantity";
+        value = 3;
+    }
+
+    @Test
+    public void shouldReturnNothingWhenMedicationCreated() {
+        service.createMedication(owner, petName, dateName, MedicationEntity);
+        verify(MedicationDao).createMedication(isA(String.class), isA(String.class), isA(String.class), isA(MedicationEntity.class));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line is longer than 120 characters (found 133).](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5348142)